packages/dashboards/src/lib/components/widget/widget-header/widget-header.component.ts

```diff
@@ -151,7 +151,7 @@ export class WidgetHeaderComponent implements OnInit, OnDestroy, AfterViewInit {
             }
             return true;
         }
-        return false;
+        return !!this.url;
```

Comment:
I'm worried about side effects in the 'other' branch. 
we can return true in both cases because it is as you mentioned: 'won't be rendered if there is no static value'